README.md

```diff
@@ -2124,13 +2124,99 @@ the build will fail if test coverage is below 100%.
 
 You should be able to do the typical stuff:
 
+### 1. Cloning repository:
+Fork the repository and run this(Only for SSH):
+```
+git clone git@github.com:${your_github_username}/foobara.git
+```
+Create a new branch for you to push into(branch name is not limited to this format)
+```
+git checkout -b develop/${your_github_username}
+```
+Now navigate to project directory
+
 ```
-git clone git@github.com:foobara/foobara
 cd foobara
+```
+### 2. Installing bundler:
+Bundler is the package manager for ruby like npm and pip are for JS and python respectively
+
+To install bundler run:
+```
+sudo gem install bundler
```

Comment:
This step shouldn't be necessary or even really possible at this point so we should remove this step